src/main/java/fr/xephi/authme/settings/SpawnLoader.java

```diff
@@ -261,6 +313,25 @@ private static boolean containsAllSpawnFields(FileConfiguration configuration, S
         return true;
     }
 
+    /**
+     * Return whether the file configuration contains all fields necessary to define a spawn
+     * under the given path.
+     *
+     * @param configuration The file configuration to use
+     * @param pathPrefix    The path to verify
+     *
+     * @return True if all spawn fields are present, false otherwise
+     */
+    private static boolean containsAllSpawnFieldsUpper(FileConfiguration configuration, String pathPrefix) {
```

Comment:
Would you agree to renaming this and the above method from `Upper` to something with `Cmi` in it? I had to read the code and think about it for a while before I understood what `upper` meant :P